aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -10,12 +10,17 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/arn"
 	"github.com/aws/aws-sdk-go/service/firehose"
+	"github.com/hashicorp/terraform/helper/customdiff"
 	"github.com/hashicorp/terraform/helper/hashcode"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/hashicorp/terraform/helper/validation"
 )
 
+const (
+	FirehoseDeliveryStreamStatusDeleted = "DESTROYED"
```

Comment:
Is there a particular reason why this constant is exported? This project is not intended for usage as a library and offers no Go compatibility guarantees. 😄 

```suggestion
	firehoseDeliveryStreamStatusDeleted = "DESTROYED"
```